wavelength-plugins/profile.js

```diff
@@ -499,6 +499,20 @@ exports.commands = {
 		"/nature delete - Removes your Profile Nature.",
 	],
 
+	'!lastactive': true,
+	checkactivity: 'lastactive',
+	lastactive: function (target, room, user) {
+		if (!target) target = user.userid;
+		if (target.length > 18) return this.errorReply("Usernames cannot exceed 18 characters.");
+		if (!this.runBroadcast()) return;
+		let targetUser = Users.get(target);
+		if (!Users(targetUser) || !targetUser.connected) return this.errorReply(`${target} is not online. Use /seen to find out how long ago they left.`);
+		if (targetUser.connected) {
```

Comment:
Don't check for `targetUser.connected` here; you just verified that it must be true if it passed `if (!targetUser || targetUser.connected)` the line before, and the only way it could possibly change would be if the user object was somehow deleted in the split second during execution, which is almost guaranteed to never happen.